### PR TITLE
Slovak_Translation_Aneta_Laukova

### PR DESCRIPTION
**Pull request by alpita-masurkar**
## Description

Description of what the PR does, includeing a fixed issue with Resolves #{issue number}

## How to test

Description of how to validate or test this PR

### CODE REVIEW

File: app/locales/sk.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
+{
+  "label": {
+    "about_title":"O",
```
Comment (tstirrat) [inline app/locales/sk.json:3]: This look incorrect.

File: app/locales/sk.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    "authorities_removal_alert_desc":"Si si istý/á, že chceš odstrániť zdroj údajov tohto úradu?",
+    "authorities_removal_alert_proceed":"Pokračovať",
+    "authorities_removal_alert_title":"Odstrániť úrad",
+    "authorities_title":"Dôveryhodné zdroje",
+    "choose_provider_subtitle":"Na to, aby si bol/a informovaný/á o tvojom vystavení sa riziku, je potrebné prihlásiť sa na odber od zdravotného úradu.",
+    "choose_provider_title":"Vybrať zdravotný úrad",
+    "commitment":"Záväzok",
+    "commitment_para":"Safe Paths bezpečne zaznamenáva a kontroluje tvoj prienik s ľuďmi, ktorí používajú tvoju polohu. Tvoje dáta NIKDY neopustia tvoj telefón bez tvojho súhlasu.",
+    "default_news_site_name":"Správy o Safe Paths",
+    "default_news_site_url":"",
```
Comment (tstirrat) [inline app/locales/sk.json:19]: remove row

File: app/locales/sk.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    "event_history_title":"História vystavenia riziku",
+    "export_para_1":"Ak si testovaný/á pozitívne na COVID-19, prosím, prispej zdieľaním svojej polohy so všetkými miestnymi orgánmi.",
+    "export_para_2":"Poloha je zdieľaná ako jednoduchý zoznam časov a miest, bez akýchkoľvek ďalších informácií.",
+    "home_at_risk_header":"Môžeš byť vystavený/á riziku",
+    "home_at_risk_subsubtext":"Toto znamená, že nie si nakazený/á",
+    "home_at_risk_subtext":"Na základe histórie tvojho GPS, je možné, že si bol/a v kontakte alebo blízko niekoho, komu bol diagnostikovaný COVID-19",
+    "home_enable_location":"Povoliť údaje o polohe",
+    "home_mayo_link_heading":"Viac informácií o COVID-19",
+    "home_mayo_link_label":"z Mayo Clinic",
+    "home_mayo_link_URL":"",
```
Comment (tstirrat) [inline app/locales/sk.json:30]: remove entry

File: app/locales/sk.json [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    "push_at_risk_message":"Stretol/la si sa s pacientom COVID-19",
+    "push_at_risk_title":"Môžeš byť v ohrození",
+    "see_exposure_history":"",
+    "settings_title":"Panel",
+    "share_location_data":"Zdieľaj svoje údaje o polohe ",
+    "show_overlap":"Kliknutím sa zobrazí verejný súbor údajov",
+    "team":"Tím",
+    "team_para":"Náš tím je zložený z konzorcia epidemiológov, inžinierov, vedcov, evanjelistov v oblasti digitálneho súkromia, profesorov a výskumníkov z uznávaných inštitúcií zahŕňajúc: MIT, Hardward, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young a Link Ventures.",
+    "terms_of_use":"Podmienky používania",
+    "terms_of_use_url":"",
```
Comment (tstirrat) [inline app/locales/sk.json:98]: remove entry

File: app/locales/sk.json [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    "tested_positive_subtitle":"Tvoje osobné údaje môžu byť odovzdané zdravotným úradom, zálohované alebo inak zdieľané.",
+    "tested_positive_title":"Zdieľaj históriu tvojej lokalizácie"
+  },
+  "history": {
```
Comment (tstirrat) [inline app/locales/sk.json:102]: Remove the whole history block, otherwise it won't fall back to english

File: app/locales/sk.json (hunk 1 above)
Comment (alpita-masurkar) [reply app/locales/sk.json:3]: I don't think so. This word was around the last time too. i had checked this. O is about in slovak :)

Comment (alpita-masurkar) [reply app/locales/sk.json:3]: will check with another native speaker too just to confirm.

Comment (tstirrat) [reply app/locales/sk.json:3]: ! cool. I learn :)

**Pull request merged**